rollup.config.js

```diff
@@ -0,0 +1,63 @@
+import typescript from '@rollup/plugin-typescript';
+import json from '@rollup/plugin-json';
+import shebang from '@walrus/rollup-plugin-shebang';
+import multiInput from 'rollup-plugin-multi-input';
+import fs from "fs";
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import pkge from './package.json'
+
+let filepaths = [];
+
+const walk = (root) => {
+  let currentPaths = fs.readdirSync(root);
+  for (let path of currentPaths) {
+    const abspath = `${root}/${path}`;
+    const isFile = /.*\.(ts|js|json)$/.test(path);
+
+    if (isFile) {
+      filepaths.push(abspath);
+    } else {
+      // isDirectory
+      walk(abspath);
+    }
+  }
+
+}
+
+walk('./src/commands');
+filepaths.push("./src/index.ts");
+
+// exclude type and test files
+filepaths = filepaths
+  .filter(path => !path.includes("test"))
+  .filter(path => !path.includes("type"));
+
+// console.log(filepaths);
+
+// exclude all dependencies from being bundled together, except for the problematic jsonLd, used by "open-attestation", "oa-verify" dependencies
+let deps = [...Object.keys(pkge.dependencies), ...Object.keys(pkge.devDependencies)]
```

Comment:
devdependencies shouldnt be bundled so let's not exclude

Reply:
All [3rd-party dependencies, i.e. node modules, shouldn't be bundled together](https://www.mixmax.com/engineering/rollup-externals). This is so regardless of whether they are direct dependencies or dev-dependencies.

Here i am trying to prevent the rollup from bundling all the node modules (except for the open-attestation libs as we want the problematic import and export lines in the esm files to be removed). In the lines below, the `deps` variable is specified in the `external` config property to exclude them

Reply:
DEV dependencies thanks

Reply:
Also we should actually bundle everything as it's not a library but an app

Reply:
remove pkge.devDependencies